torch/_meta_registrations.py

```diff
@@ -3969,8 +3971,20 @@ def meta__scaled_dot_product_efficient_backward(
         dtype=value.dtype,
         device=value.device,
     )
-
-    return grad_q, grad_k, grad_v
+    grad_bias = torch.empty()
+    if grad_bias.requires_grad:
+        assert isinstance(attn_bias, TensorLike)
+        nominal_sizes = attn_bias.size()
+        last_dim_size = nominal_sizes[-1]
+        alignTo = 16
+        nominal_sizes[-1] = alignTo * ((last_dim_size + alignTo - 1) / alignTo)
+        grad_bias = torch.empty(
+            nominal_sizes,
+            dtype=attn_bias.dtype,
+            device=attn_bias.device,
+        )[:, :, :, :last_dim_size]
```

Comment:
this can be wrong unless we move the 4d expand to the top level sdpa where it is composite